.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Default
```

Comment:
Can you add `@ramfox`? They started the repo

Reply:
yep. and will change our names to the team.